### PR TITLE
feat(ngx-conveyer): reduce view updates and fix outputs

### PR DESCRIPTION
**Pull request by arturovt**
Had to remove `@Directive -> outputs` and switch to the explicit declaration because currently the template compiler is not happy. Event emitters also should be declared on the class when `outputs` is provided so it's actually easier to use the explicit `@Output(...)` declaration:

Screenshot from 2023-03-11 20-05-58

### CODE REVIEW

**Comment by malangfox** [comment]
Hello @arturovt.

We appreciate your contributions to our components!

The outputs related issue you left seems to be an issue with a module we use.
I think you can keep the original code by modifying `ngx-conveyer.interface.ts` as follows.

Before
```
import { REACTIVE_CONVEYER } from '@egjs/conveyer';
import { AngularReactiveAdapterResult } from '@cfcs/angular';


export class NgxConveyerInterface { }

// automatic reactive state, methods types
export interface NgxConveyerInterface
  extends AngularReactiveAdapterResult<typeof REACTIVE_CONVEYER> { }
```

After
```
import { REACTIVE_CONVEYER } from '@egjs/conveyer';
import { ReactiveAdapterResult } from '@cfcs/angular';


export class NgxConveyerInterface { }

// automatic reactive state, methods types
export interface NgxConveyerInterface
  extends ReactiveAdapterResult<typeof REACTIVE_CONVEYER, "ngx"> { }
```

We will fix this issue above. We'll also be happy to review the changes you've made.
Thank you very much!

**Comment by arturovt** [comment]
Alright, I'll revert my outputs updates and will add the `ngx` prefix.

**Review by malangfox — APPROVED** [review]
LGTM

**Review by daybrush — APPROVED** [review]
LGTM

File: packages/ngx-conveyer/README.md
Comment (WoodNeck) [inline packages/ngx-conveyer/README.md:1]: Nice retouch :)

**Review by WoodNeck — APPROVED** [review]
LGTM

**Pull request merged**